src/lib/structures/SlashCommandStore.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * This is the most basic of examples for a store, this is all you really need
+ * to actively register a store with Sapphire, and anything in the supplied
+ * folder will load if it extends a Sapphire piece.
+ */
+const SlashCommand = require("./SlashCommandPiece.js");
+const { Store } = require("@sapphire/framework");
+
+module.exports = class SlashCommandStore extends Store {
+  constructor() {
+    // This is the name of the directory we want to look in for our slash
+    // commands.
+    super(SlashCommand, { name: "slashCommands" });
+  }
+
+  async registerCommands() {
```

Comment:
Moved the deploySlashCommands script here, removed the try catch so it can be handled by the trigger. See `ready.js` (added try/catch there)